app/Assets/Helpers.php

```diff
@@ -110,7 +113,7 @@ public function getExtension(string $filename, bool $isURI = false): string
 		// https://github.com/electerious/Lychee/issues/482
 		list($extension) = explode(':', $extension, 2);
 
-		if (empty($extension) === false) {
+		if (!empty($extension)) {
```

Comment:
Shouldn't be a simple fix for the error
```suggestion
		if ($extension !== '') {
```
? If we remove it anyway with the filestreams PR, I wouldn't say this is something that should be a blocker.